modules/programs/rofi-pass.nix

```
@@ -34,13 +35,25 @@ in {
         <https://github.com/carnager/rofi-pass/blob/master/config.example>.
       '';
     };
+
+    package = mkOption {
+      default = pkgs.rofi-pass;
+      type = types.package;
+      description = ''
+        Package providing the {command}`rofi-pass` binary.
+        Useful for when you're on wayland
+      '';
+      example = literalExpression ''
+        pkgs.rofi-pass-wayland
+      '';
+    };
```

You could simplify this to

```suggestion
    package = mkPackageOption pkgs "rofi-pass" {
      example = "pkgs.rofi-pass-wayland";
    };
```